### DISCUSSION TITLE
交易之难 # 组合季报(2022Q2) · BMPI

### DISCUSSION THREAD

**Discussion by giscus** [bmpi-dev/bmpi.dev#115] [Announcements]
# 交易之难 # 组合季报(2022Q2) · 构建我的被动收入

被动收入投资组合2022年Q2季报

https://www.bmpi.dev/money/passive-income-protfolio/2022q2/

**Comment by xk822677236** [comment]
双均线策略反复出现买入卖出信号确实难受😣。单边向上或单边向下使用双均线是最舒服的。
不过中概互联和医药近期太弱拖累整体收益。
之前还以为出现卖出信号，会做清仓处理。看当前持仓主理人可能只是根据估值做相应减仓。
